src/s3-resource-selector/s3-modal/basic-table.tsx

```diff
@@ -172,3 +187,38 @@ export function BasicS3Table<T>({
     />
   );
 }
+
+interface InternalHeaderActionsProps<T> {
+  reloadData: () => void;
+  i18nStrings: BasicS3TableProps<T>['i18nStrings'];
+  lastUpdated: Date | undefined;
+}
+
+export function InternalHeaderActions<T>({ i18nStrings, reloadData, lastUpdated }: InternalHeaderActionsProps<T>) {
+  function getLastUpdated() {
+    if (!lastUpdated || !i18nStrings.lastUpdatedText) {
+      return null;
+    }
+
+    const formattedDate = formatDateLocalized({
```

Comment:
The examples in our date formatting guidelines use month-first format because they are assume "en-US" locale. `formatDateLocalized` uses the browser native `Intl.DateTimeFormat` API to generate the formatted dates, which will use month-first format for "en-US" and day-first format for "en" or "en-UK".

If you set the locale to "en-US" to the [date range picker with absolute human-readable format](https://cloudscape.design/components/date-range-picker/?tabId=playground&example=with-absolute-human-readable-format), it will show the month-first format.

Reply:
Long story short: the apparent mismatch with the documentation is known and only because of different locale. I think `formatDateLocalized` works correctly and that we should use it here in the same way.